modules/cnf-cpu-infra-container.adoc

```diff
@@ -5,17 +5,30 @@
 [id="cnf-cpu-infra-container_{context}"]
 = Restricting CPUs for infra and application containers
 
-You can reserve cores (threads) from a single NUMA node for operating system housekeeping tasks and put your workloads on another NUMA node. Partitioning the CPUs this way can prevent the housekeeping processes from impacting latency-sensitive application processes. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. 
+Housekeeping and workloads may utilize the central processing units (CPUs) in a way that would impact latency sensitive processes running on those CPUs. Partitioning your workloads on separate CPUs prevents the processes from interfering with each other. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. 
 
-You can ensure that housekeeping tasks and workloads run on separate NUMA nodes by specifying two groups of CPUs in the `spec` section of the performance profile. 
+The exact partitioning pattern to use depends on many factors like hardware, workload characteristics and system load. Some sample use cases are as follows:
+
+* If the latency sensitive workload uses specific hardware (like network interface cards (NICs)), ensure that the CPUs in the isolated pool are as "close" to this hardware as possible. Place the workload in at least the same Non-Uniform Memory Access (NUMA) node.
+
+* The reserved pool is used for handling all interrupts, but not all interrupt requests generate noise. When depending on system networking, allocate a sufficiently sized reserve pool to handle all the incoming packet interrupts.
+
+The above examples require detailed analysis and measurements to be combined into a pattern where all housekeeping CPUs are allocated from a single NUMA node. The reserved CPUs can also be distributed among all NUMA nodes.
```

Comment:
Housekeeping can also span multiple NUMA nodes and quite commonly do. There really are both options and it the selection depends on the workload architecture and the specific user use case.

Reply:
Updated as requested.